fe/src/main/java/org/apache/doris/analysis/CreateRoutineLoadStmt.java

```diff
@@ -326,6 +335,16 @@ private void checkKafkaCustomProperties() throws AnalysisException {
                 }
             }
         }
+        // check offsets
+        // Todo(ml)
+        final String kafkaOffsetsString = customProperties.get(KAFKA_OFFSETS_PROPERTY);
+        if (kafkaOffsetsString != null) {
+            kafkaOffsets = new ArrayList<>();
+            String[] kafkaOffsetsStringList = customProperties.get(KAFKA_OFFSETS_PROPERTY).split(",");
+            for (String s : kafkaOffsetsStringList) {
+                kafkaOffsets.add(Long.valueOf(s));
```

Comment:
You should check the number format. And also check whether the number of specified offsets equals to number of specified partitions

Reply:
I will do it later